src/openads/application/service/DisplayPositionUseCase.js

```diff
@@ -29,4 +34,18 @@ export default class DisplayPositionUseCase {
     }
     return optionalPositionWithId.position
   }
+
+  _filterPositionAdAvailable (position) {
+    return position.ad
+      .then(adResponse => adResponse.status)
+      .then(status => AD_AVAILABLE === status)
+      .then(available => available ? position : Promise.reject(new PositionAdNotAvailableError({id: position.id})))
+  }
+
+  _filterPositionAdIsNative (position) {
+    return position.ad
+      .then(adResponse => adResponse.data.adType)
+      .then(adType => adType === 'native')
```

Comment:
There is a file in infrastructure/appnexus/event which includes all possible events from appnexus, we should do the same with "adType" and use that constant here ;)

Reply:
Done!

Reply:
Done!